content/en/security_platform/cloud_siem/log_detection_rules.md

```diff
@@ -206,6 +245,26 @@ Once a signal is generated, the signal will remain "open" if the data remains an
 
 A signal will "close" regardless of whether or not the anomaly is still anomalous once the time exceeds the maximum signal duration. This time is calculated from the first seen timestamp.
 
+[1]: /monitors/notify/?tab=is_alert#integrations
+{{% /tab %}}
+
+{{% tab "Impossible Travel" %}}
+
+The impossible travel detection method does not require setting a rule case.
+
+### Severity and notification
+
+Select an appropriate severity level for the security signal (`INFO`, `LOW`, `MEDIUM`, `HIGH`, `CRITICAL`).
+
+In the “Notify” section, configure zero or more [notification targets][1].
+
+### Time windows
+
+An `evaluation window` is specified to match when at least one of the cases matches true. This is a sliding window and evaluates in real-time.
```

Comment:
I know this is the same phrasing as for the other rule types but `matches true` seems off to me.